analyzer/src/main/resources/init.gradle

```diff
@@ -125,15 +125,24 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
     }
 
     private static class DependencyTreeModelBuilder implements ToolingModelBuilder {
+        static DependencyImpl dependencyFromDisplayName(String displayName, List<Dependency> dependencies,
+                                                        String error) {
+            if (displayName.startsWith('project :')) {
+                def coordinates = displayName.split(':', 2)
+                return new DependencyImpl(groupId: '<project>', artifactId: coordinates[1], dependencies: dependencies,
+                        error: error.toString())
+            }
+
+            def coordinates = displayName.split(':', 3)
```

Comment:
There could possibly be more kinds of displayNames, maybe created by plugins. To be on the safe side we should verify here that `coordinates.size() == 3` and otherwise do something like`new DependencyImpl(groupId: '<unknown>', artifactId: displayName, dependencies: dependencies, error: error.toString())`.

Reply:
Done.